### PR TITLE
add mc2 to vagrant machine

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by smn** [comment]
the way I was thinking about this was a local environment for when we're doing dev on mission control on the host machine, not necessarily something we'd want MC to run inside of.

Happy to hear your reasons for putting it inside though! And if we do, can we get it to be a `pip install` rather than a source checkout? I feel like if we go for a source checkout now we're creating opportunity for us to be lazy and never get it `pip install`-able.

**Comment by JayH5** [comment]
If MC is going to be part of every seed stack deployment as well as being the primary way to interact with the rest of the stack then I'd say it should be bundled in this VM.

I am, however, with @smn on getting it `pip install`-able. We need to work on reducing the size of this image and that potentially means removing things like git.

File: provision/install.sh [hunk 1]
```diff
@@ -53,6 +53,32 @@ pip install consular
 mkdir -p /tmp/consul
 mkdir -p /usr/share/consul
 
+# Install MC2
+
+apt-get install -y \
+    libpq-dev \
+    libffi-dev \
```
Comment (JayH5) [inline provision/install.sh:60]: Already installed for consular.

**Comment by JayH5** [comment]
@miltontony: I'm closing this for now since things have changed so much. Please see praekeltfoundation/puppet-seed_stack#31. We need to make the change in that repo.

**Pull request closed without merging**